controls/tests.py

```diff
@@ -25,7 +25,9 @@
 from system_settings.models import SystemSettings
 from .models import *
 from .oscal import Catalogs, Catalog
-
+from .models import *
```

Comment:
It is best practice to import the specific models that you need instead of using `*`

Reply:
Using specific models imported on `siteapp.models`. Commited on [
f68a670](https://github.com/GovReady/govready-q/pull/1203/commits/f68a670bd2a277b0632fe3a46e5af8dbd5678b39)